src/meta/node/src/lib.rs

```diff
@@ -83,9 +83,6 @@ pub struct MetaNodeOpts {
     #[clap(long, env = "RW_SQL_ENDPOINT")]
     pub sql_endpoint: Option<String>,
 
-    #[clap(long, env = "RW_DASHBOARD_UI_PATH")]
-    pub dashboard_ui_path: Option<String>,
```

Comment:
Does cloud use such env var?

Reply:
I don't think so. We've already set it in Dockerfile. 😄